src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
```

Comment:
Please remove the redundant empty line after the class declaration (line 2). According to the checklist, there should not be an empty line after the class declaration.